definitions/npm/bem-cn_v3.x.x/flow_v0.90.x-/bem-cn_v3.x.x.js

```diff
@@ -0,0 +1,33 @@
+declare module 'bem-cn' {
+  declare type Mod = { [mod: string]: string | boolean };
+  declare type State = { [state: string]: boolean };
+  declare type Mix = string | string[] | BlockItem;
+  declare type SetupOptions = {|
+    ns?: string,
+    el: string,
+    mod: string,
+    modValue: string,
+    classMap?: { [string]: string },
+  |};
+
+  declare type BlockItem = {
+    (): string,
```

Comment:
Oh nice I didn't realize this was valid syntax.

Reply:
Yeah, me too :)